.github/workflows/version-changelog-update.yml

```diff
@@ -0,0 +1,39 @@
+name: Version Bump and Changelog Update
+
+on:
+  push:
+    branches:
+      - releases
+    paths-ignore:
+      - package.json
+      - CHANGELOG.md
+
+jobs:
+  version-changelog-update:
+    permissions:
+      contents: write
```

Comment:
How about we require the tests to pass? I believe the syntax would be:

```diff
 jobs:
+  tests:
+    uses: ./.github/workflows/runtest.yml
   version-changelog-update:
+    needs: tests
     permissions:
       contents: write
```

Along with [a corresponding addition of `workflow_call` to `runtest.yml`](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow).

Reply:
I like the thought. Seems doing something like the following in `runtest.yml` to avoid doubling up on that run should be done:

```yml
# runtest.yml

on:
  push:
    branches-ignore:
      - releases # version-changelog-update which triggers on a push to releases will still trigger this workflow through `workflow_call`
  pull_request:
  workflow_call:
```

Seems useful as well on #912, for an automated deployment.